src/core/bittorrent/torrenthandle.cpp

```diff
@@ -1055,12 +1055,12 @@ int TorrentHandle::downloadPayloadRate() const
     return m_nativeStatus.download_payload_rate;
 }
 
-int TorrentHandle::totalPayloadUpload() const
+qlonglong TorrentHandle::totalPayloadUpload() const
```

Comment:
There is no technical need, it is just more easy to understand that this is 64bit integer.
So to improve code read-ability.
But you might as well not to bother about it.

@naikel Please do not worry about what types are used on other places.
`qlonglong` and `qint64` is the same.
Do what you think right.